yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -65,8 +72,20 @@ def _real_extract(self, url):
         formats = []
         for video_file in video_files:
             v_url = video_file.get('url')
+            v_type = video_file.get('type')
             if not v_url:
                 continue
+            if v_type == 'application/deferred':
+                d_param = urllib.parse.quote(v_url)
+                headers = {'Authorization': 'Bearer ' + video_file.get('token')}
+                json = self._download_json(
+                    f'https://api.tv5monde.com/player/asset/{d_param}/resolve?condenseKS=true', v_url,
+                    note='Downloading deferred info', headers=headers)
+                v_url = json[0]['url']
```

Comment:
- make everything non-fatal, use `traverse_obj` (import from `utils.traversal`)
- do not use `json` as var name
- import `url_or_none` from `utils`
```suggestion
                token = video_file.get('token')
                if not token:
                    continue
                deferred_json = self._download_json(
                    f'https://api.tv5monde.com/player/asset/{d_param}/resolve?condenseKS=true', display_id,
                    note='Downloading deferred info', headers={'Authorization': f'Bearer {token}'}, fatal=False)
                v_url = traverse_obj(deferred_json, (0, 'url', {url_or_none}))
                if not v_url:
                    continue
```